include/seqan3/argument_parser/validators.hpp

```diff
@@ -561,9 +566,8 @@ class input_file_validator : public file_validator_base
     //!\brief Returns a message that can be appended to the (positional) options help page info.
     std::string get_help_page_message() const
     {
-        return detail::to_string("Valid input file formats: [",
-                                 extensions | views::join(std::string{", "}),
-                                 "]");
+        return "The input file must exist and read permissions must be granted. " +
```

Comment:
I am not sure. Is that really necessary with the permissions? That kind of clutters the output. It is clear that you need to have permissions. If not you will get an exception saying that you don't have permissions. 
I think if the description reads: `An input file with a valid extension: [ext1, ext2, ...]`
The other thing is `An output file with a valid extension: [ext1, ext2, ...]`
Then the code is 
```cpp
return valid_extensions_help_page_message("input file");
```
And the help page message assembles the string.
Then also it checks if valid extensions is empty and omits the part with the valid extensions. Also note, that we should talk about extensions and not formats.

Reply:
> I am not sure. Is that really necessary with the permissions? That kind of clutters the output. It is clear that you need to have permissions.

Well, although "it is obvious that you need them", it is a feature we provide to check for this and I thought this documentation helps to indicate this?

And whatever we decide about the permissions, I think it is important to note that the output file must not exist. This is not the usual behaviour by default as @joergi-w mentioned in his recent issue.

Reply:
> And whatever we decide about the permissions, I think it is important to note that the output file must not exist. This is not the usual behaviour by default as @joergi-w mentioned in his recent issue.

Mhmm, I think it is not necessary but I have no strong feelings. But the other points should be adresed. `valid formats` ➡️ `valid extension` and if the extension set is empty this string should not be displayed at all.